### ISSUE TITLE
Research ways to improve model accuracy

### ISSUE THREAD

**Issue by ShwetanshSingh** [ShwetanshSingh/speech_emotion_recognition_model#8]
The model does not give satisfactory accuracy on test set (~50%), even when changing epochs in training.
Must look for improvements in method of training.

Maybe add more data
Change augmentation techniques
Use different features from audio
Use a pretrained model from HuggingFace
